src/lib/parsers/Args.ts

```diff
@@ -205,7 +203,11 @@ export class Args {
 		if (this.parser.finished) return this.missingArguments();
 
 		const state = this.parser.save();
-		const data = this.parser.many().reduce((acc, token) => `${acc}${token.value}${token.trailing}`, '');
+		const data = this.parser
+			.many()
+			.unwrapOr<Parameter[]>([])
+			.map((parameter) => parameter.value)
```

Comment:
This should use `parameter.leading`, otherwise you're breaking many things in many ways, e.g. `eval` command suddenly losing all of the spaces.